doc/src/manual/mathematical-operations.md

```diff
@@ -457,7 +457,7 @@ Juxtaposition parses like a unary operator, which has the same natural asymmetry
 Julia supports three forms of numerical conversion, which differ in their handling of inexact
 conversions.
 
-  * The notation `T(x)` or `convert(T,x)` converts `x` to a value of type `T`.
+  * The notation `T(x)` or `convert(T, x)` converts `x` to a value of type `T`.
```

Comment:
This text isn't strictly true for a constructor. Perhaps only `convert` should be mentioned here?

Reply:
I don't quite understand what you mean here.